microcosm_sqlite/tests/test_threading.py

```diff
@@ -0,0 +1,45 @@
+"""
+Threading tests.
+
+"""
+from multiprocessing.pool import ThreadPool
+from tempfile import NamedTemporaryFile
+
+from hamcrest import assert_that, contains
+from microcosm.api import create_object_graph
+from microcosm.loaders import load_from_dict
+
+from microcosm_sqlite.context import SessionContext
+from microcosm_sqlite.stores import GetOrCreateSession
+from microcosm_sqlite.tests.fixtures import Example, Person, PersonStore
+
+
+def test_threading():
```

Comment:
SQLite resources can't be used across multiple threads. That means that if we don't want to declare a context every time we use a store, we need to lazy init the session within each thread.

This works now.